hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSpark.java

```diff
@@ -257,6 +257,7 @@ private static SparkSession initializeSparkSession() {
         // Set default warehouse directory, currently spark can get confused.
         // User Supplied splice.spark.sql.warehouse.dir will overwrite it.
         conf.set("spark.sql.warehouse.dir", "/user/splice/spark-warehouse");
+        conf.set("spark.sql.autoBroadcastJoinThreshold", "-1");
 
```

Comment:
Why disable broadcast join be default?

Reply:
If we don't disable this, Spark's catalyst might decide that some join would be better as a broadcast join instead of a normal mergesort/hash join. Since we want our optimizer to choose when to do broadcasts, I have to disable it.